dipy/testing/decorators.py

```diff
@@ -73,3 +73,18 @@ def test_with_xvfb(*args, **kwargs):
     # Plant it back in and return the new function:
     test_with_xvfb.__name__ = fname
     return test_with_xvfb if not is_windows else my_test
+
+
+def set_random_seed(func):
+    """Decorator to set equal seeds.
+
+    This will help us avoid random errors
+    that occur with low probabilitiy during testing.
+
+    """
+    def test_with_seed(*args, **kwargs):
+        np.random.seed(2023)
```

Comment:
This has been deprecated in favor of:
```
rng = np.random.default_rng(<seed_value>)
rng.random(<value_count>)
```

so the decorator should be changed. Not sure how hard it is to make the `rng` variable be exposed to the testing method.

https://numpy.org/doc/stable/reference/random/generator.html
https://numpy.org/neps/nep-0019-rng-policy.html

Reply:
Yeah I understand it is deprecated, but as you said I am not sure how to use the decorator with the rng variable. Alternatively, we could just change each random function individually.

Reply:
The risk of using a deprecated value is the removal in a future `NumPy` version, so better IMO it is better to stick to the current convention now.

Reply:
I did not know that `np.random.seed` has been deprecated. Thank you for the info @jhlegarreta .

We need to update that in the whole code based. I prefer the new standard.